scripts/corr2_dsim_control.py

```diff
@@ -48,6 +48,8 @@
                     'Choose which Output to source from, Output_0 or Output_1.\
                      Output Scale, choose between 0 - 1.\
                      Output types, choose from signal or test_vectors.')
+parser.add_argument('--sins', action='append', default=[], nargs=2, help=
```

Comment:
I don't think we need to add a new argument. Just modify the existing --sine-source and --noise-source and --select-output arguments to handle multiple arguments. In otherwords, they should be changed into a loop that loops over the settings for each source.